go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/multiformats/go-multihash v0.1.0
 	github.com/onflow/atree v0.1.1
 	github.com/onflow/cadence v0.21.0
-	github.com/onflow/flow v0.2.3-0.20220131193101-d4e2ca43a621
+	github.com/onflow/flow v0.2.3-0.20220207221328-d1ffd1c3053e
```

Comment:
latest on master on the [onflow/flow repo](https://github.com/onflow/flow/commit/d1ffd1c3053eaf3f5386b20af44911b5b59678c7)